deploy.rb

```diff
@@ -10,7 +10,7 @@ def on_standby?
   met? {
     output = raw_shell('npm ls', :cd => path)
     # Older `npm` versions exit 0 on failure.
-    output.ok? && output.stdout['UNMET DEPENDENCY'].nil?
+    output.ok? && ['UNMET DEPENDENCY', '(empty)'].none? { |str| output.stdout[str] }
```

Comment:
This is where my lack of node experience rears it's head, but I believe the name `devDependencies` is more like "non-runtime" dependencies (when you consider node running as a server). Things like webpack are non-runtime dependencies, as they shouldn't be running in a node server context.

Another aspect of this is that ideally we wouldn't have to run webpack on the production servers at all, but it's easier at the moment because it fits into our current deploy methods.

If you consider a full CI setup, I could see the assets being compiled on one of the tanks as the part of the final build step before deploying, then uploading those assets on deploy. In that situation, having webpack and friends as `devDependencies` really makes sense.